nx-X11/programs/Xserver/GL/glx/indirect_dispatch.c

```diff
@@ -25,7 +25,7 @@
  * SOFTWARE.
  */
 
-#include <X11/Xmd.h>
+#include <nx-X11/Xmd.h>
```

Comment:
@uli42: shouldn't this commit (48a080b384fc8168eddae8542990ed9e181de96e) be pulled over to the small_fixes PR #532 ?